_types/ComputationalWorkflow/1.0-RELEASE.html

```diff
@@ -14,7 +14,7 @@
 previous_version: "0.1-DRAFT-2020_07_21"
 previous_release:
 
-dateModified: 2020-10-19
+dateModified: 2021-03-04
```

Comment:
Should it not be 2021-03-09 as in the version date 20210309T210904 ?

Reply:
The last edit on the type was 4 March I think.

Reply:
Ok